src/vmware/CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Release History
+
+## 2.0.0 (TBD)
+
+- prompt for legal terms in private-cloud create [#2932](https://github.com/Azure/azure-cli-extensions/pull/2932)
+
+## 1.0.0 (2020-11-02)
+
+- GA Release
+- Moved to https://github.com/Azure/azure-cli-extensions
+
+## 0.7.2 (2020-10-20)
+
+- See https://github.com/Azure/az-vmware-cli/releases
+
+## 0.7.1 (2020-10-19)
+
+## 0.7.0 (2020-07-21)
+
+## 0.6.0 (2020-05-06)
+
+## 0.5.5 (2020-03-19)
+
+## 0.5.4 (2020-03-13)
+
+## 0.5.3 (2020-03-12)
+
+## 0.5.2 (2020-03-10)
+
+## 0.5.1 (2020-03-10)
+
+## 0.5.0 (2020-03-09)
```

Comment:
I think you can delete them because there is no content.

Reply:
I added content and removed the empty versions.